yapapi/executor/__init__.py

```diff
@@ -87,7 +87,7 @@ def __init__(
         max_workers: int = 5,
         timeout: timedelta = timedelta(minutes=5),
         budget: Union[float, Decimal],
-        strategy: MarketStrategy = DummyMS(),
+        strategy: MarketStrategy = LeastExpensiveLinearPayuMS(),
```

Comment:
I'd change the default strategy in a separate pull request and leave only the `max(...)` change in the current one.

Reply:
Never mind, let's do this in a single PR.